packages/ui/src/common/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+
+import { Arrow, CheckboxIcon } from '@/common/components/icons'
+import { TextInlineSmall } from '@/common/components/typography'
+import { BorderRad, Colors, Transitions } from '@/common/constants'
+import { spacing } from '@/common/utils/styles'
+
+import { StepperTheme } from './themes'
+import { asStepsToRender, StepperStep, StepToRender } from './types'
+
+export interface StepperProps {
+  steps: StepperStep[]
+  theme?: typeof StepperTheme.light
```

Comment:
I'd go with:

```suggestion
  theme?: keyof typeof StepperTheme
```

And have a default `dark`. The usage will be:

```typescript
<Stepper steps={} theme="light" />
```

instead of importing the object, the component itself would choose the proper theme style object or pass it to the styled-components, WDYT?

Reply:
Right! that looks better :+1: